objgraph.py

```diff
@@ -405,17 +408,20 @@ class name part.
     .. versionchanged:: 1.8
        New parameter: ``shortnames``.
 
+    .. versionchanged:: 1.8.2
+       New parameter: ``output``.
+
```

Comment:
It'll be 1.9.0, since I try to use semantic versioning and this is a new feature (rather than a bugfix).